packages/firebase_database/ios/Classes/FirebaseDatabasePlugin.m

```diff
@@ -25,6 +30,16 @@ - (FlutterError *)flutterError {
   return ref;
 }
 
+NSString *getTransactionKey(NSDictionary *arguments) {
+  NSString *transactionKey = arguments[@"transactionKey"];
+  return transactionKey;
+}
+
+NSInteger getTransactionTimeout(NSDictionary *arguments) {
```

Comment:
You can eliminate this function and just inline it

Reply:
Done!